pkgs/development/python-modules/pymupdf4llm/default.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  pymupdf,
+}:
+
+buildPythonPackage rec {
+  pname = "pymupdf4llm";
+  version = "0.0.17";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "pymupdf";
+    repo = "RAG";
+    tag = "v${version}";
+    hash = "sha256-+RLK+UorkU8eVQJGrc7pVNZPtIpxMgA9mBKA6GeWUa0=";
+  };
+
+  sourceRoot = "${src.name}/pymupdf4llm";
+
+  build-system = [ setuptools ];
+
+  dependencies = [ pymupdf ];
+
+  checkPhase = ''
+    python3 - <<'EOF'
```

```suggestion
  checkPhase = ''
    runHook preCheck

    python3 - <<'EOF'
```
Could you add the `runHook` when overriding the default phase? This would help downstream users who want to run `preCheck` and/or `postCheck` phases.
Reference: https://nixos.org/manual/nixpkgs/unstable/#sec-stdenv-phases

---

fixed. (thanks the reference)